proxies/live/apiproxy/proxies/default.xml

```diff
@@ -9,6 +9,11 @@
         <Name>OauthV2.VerifyAccessToken</Name>
       </Step>
     </Request>
+    <!-- <Request>
+      <Step>
+          <Name>KeyValueMapOperations.GetVariables</Name>
+      </Step>
+    </Request> -->
```

Comment:
should it be commented?

Reply:
yes, it will be uncommented when JWKS will be ready